Neos.Fusion.Afx/README.md

```diff
@@ -368,11 +368,11 @@ prototype(Vendor.Site:IterationExample) < prototype(Neos.Fusion:Component) {
 
     renderer = afx`
         <ul @if.has={props.items ? true : false}>
-        <Neos.Fusion:Collection collection={props.items} itemName="item">
+        <Neos.Fusion:Loop items={props.items} itemName="item">
```

Comment:
We don’t need the @path on the li child below I think ^^

Reply:
Good catch ... is adjusted